test/e2e/scripts/setup-test-tokens.sh

```
@@ -0,0 +1,250 @@
+
+    local current_user api_server
+    current_user=$(oc whoami)
+    api_server=$(oc whoami --show-server)
+    echo "Current admin session: $current_user (will be preserved)"
+
+    export ADMIN_OC_TOKEN=""
+    export TOKEN=""
+
+    local temp_kubeconfig
+    temp_kubeconfig=$(mktemp)
+    trap "rm -f '$temp_kubeconfig'" RETURN
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)